Kudu.Contracts/Settings/SettingsKeys.cs

```diff
@@ -35,5 +35,9 @@ public static class SettingsKeys
         public const string SiteExtensionsFeedUrl = "SCM_SITEEXTENSIONS_FEED_URL";
         public const string DisableDeploymentOnPush = "SCM_DISABLE_DEPLOY_ON_PUSH";
         public const string TouchWebConfigAfterDeployment = "SCM_TOUCH_WEBCONFIG_AFTER_DEPLOYMENT";
+        public static string CustomLogStream(string streamKey)
```

Comment:
We usually don't do it here.  Instead, we put all setting helper in [extensions](https://github.com/projectkudu/kudu/blob/master/Kudu.Contracts/Settings/DeploymentSettingsExtension.cs).